.github/workflows/docker_simulate-sce.yml

```diff
@@ -0,0 +1,78 @@
+name: Build docker image for simulate-sce
+env:
+  MODULE: simulate-sce
+  MODULE_PATH: analyses/simulate-sce
+  REGISTRY_ALIAS: openscpca
+
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - analyses/simulate-sce/Dockerfile
+      - analyses/simulate-sce/.dockerignore
+      - analyses/simulate-sce/renv.lock
+      - analyses/simulate-sce/conda-lock.yml
+  push:
+    branches:
+      - main
+    paths:
+      - analyses/simulate-sce/Dockerfile
+      - analyses/simulate-sce/.dockerignore
+      - analyses/simulate-sce/renv.lock
+      - analyses/simulate-sce/conda-lock.yml
+
+permissions:
+  id-token: write # This is required for requesting the JWT
+  contents: read # This is required for actions/checkout
+
+jobs:
+  build:
+    name: Build Docker Image
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          role-to-assume: arn:aws:iam::992382809252:role/GithubOpenId
+          role-session-name: githubActionSession
+          aws-region: us-east-1
+
+      - name: Log in to Amazon ECR
+        id: login-ecr
+        if: github.event_name == 'push'
+        uses: aws-actions/amazon-ecr-login@v2
+        with:
+          registry-type: public
+
+      - name: Create ECR repository if needed
+        id: create-ecr
+        if: github.event_name == 'push'
+        run: |
+          aws ecr-public describe-repositories --repository-names ${{ env.MODULE }} \
+          || aws ecr-public create-repository --repository-name ${{ env.MODULE }}
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Docker metadata
+        id: meta
+        uses: docker/metadata-action@v5
+        with:
+          images: public.ecr.aws/${{ env.REGISTRY_ALIAS }}/${{ env.MODULE }}
+          # tag with semver for releases latest for main branch, ref for prs (tags with branch name)
+          tags: |
+            type=raw,value=latest,enable={{is_default_branch}}
+            type=semver,pattern={{raw}}
+            type=ref,event=pr
```

Comment:
Does this mean a new image is pushed every time there is a change, and we aren't using any versioned tags?

Reply:
Okay makes sense 👍

Reply:
The real test will be if it works as expected on merge!